### ISSUE TITLE
TimescaleDB CMake Error

### ISSUE THREAD

**Issue by pauljsymonds** [PostgresApp/PostgresApp#751]
Trying to install / Build Timescale DB.   Their [docs](https://docs.timescale.com/self-hosted/latest/) state if you install Postgres by any other method than Brew that you will receive errors and point to build from source.

```
% make install
[  0%] Built target sqlfile
[ 18%] Built target sqlupdatescripts
[ 18%] Built target gitcheck
[ 66%] Built target timescaledb
[ 70%] Built target timescaledb-loader
[100%] Built target timescaledb-tsl
Install the project...
-- Install configuration: "Release"
-- Installing: /Applications/Postgres.app/Contents/Versions/16/share/postgresql/extension/timescaledb.control
CMake Error at cmake_install.cmake:49 (file):
  file INSTALL cannot copy file
  "/Users/paulsymonds/Development/timescaledb/build/timescaledb.control" to
  "/Applications/Postgres.app/Contents/Versions/16/share/postgresql/extension/timescaledb.control":
  Operation not permitted.


make: *** [install] Error 1
```

Apple have clearly locked this down

There is a way though you can allow Terminal Access in Settings!

see link here also https://github.com/timescale/timescaledb/issues/6474

**Comment by jakob** [comment]
> There is a way though you can allow Terminal Access in Settings!

Yes, I've discovered this previously, it was discussed in #719. We should probably document that somewhere

**Comment by jakob** [comment]
I checked and t's already documented in the docs https://postgresapp.com/documentation/install.html in the section titled "Building custom PostgreSQL extensions".

However, that page is pretty long. If you have suggestions how to make the docs more accessible, please let us know.

**Comment by jakob** [comment]
The TimescaleDB extenstion for Postgres.app is now available for download: https://postgresapp.com/extensions/

I've also updated the instructions for building extensions -- see the bottom of the linked page!
